src/gui/guiAnimatedImage.h

```diff
@@ -4,7 +4,7 @@
 #include <IGUIElement.h>
 #include <IGUIEnvironment.h>
 
-using namespace irr;
+
```

Comment:
Not very important but in this file and few other the removed line leave 3 empty lines in a raw, 1 or 2 empty lines is enough after the includes

Reply:
Sure, I've now trimmed this to 2 blank lines after includes everywhere: https://github.com/luanti-org/luanti/pull/16309/commits/0de2d75b4297bd18683b5fa77a130888d5806998

This also affects some other files but I think that's a feature.